h2o-core/src/main/java/hex/grid/GridSearchStrategy.java

```diff
@@ -0,0 +1,5 @@
+package hex.grid;
+
+public enum GridSearchStrategy {
+  SEQUENTIAL, PARALLEL
```

Comment:
no need to shout :)

Reply:
This is actually forgotten and obsolete from times when there were two parameters (strategy and degree of parallelism usable in parallel mode). - unless we want to implement two arguments for the users.

Reply:
I am fine with keeping the py/R api int based but internally I would use something more high-level